packages/performance-tests/test/iModelUtils.ts

```diff
@@ -0,0 +1,82 @@
+import * as fs from "fs";
+import * as path from "path";
+import { ElementGroupsMembers, IModelDb, IModelHost, PhysicalModel, PhysicalObject, SpatialCategory, StandaloneDb } from "@itwin/core-backend";
+import { Guid, OpenMode } from "@itwin/core-bentley";
+import { BriefcaseIdValue, Code } from "@itwin/core-common";
+import { initOutputFile } from "./TestUtils";
+import { Point3d, YawPitchRollAngles } from "@itwin/core-geometry";
+import { IModelTransformerTestUtils } from "@itwin/imodel-transformer/lib/cjs/test/IModelTransformerUtils";
+import { getTShirtSizeFromName, TestIModel  } from "./TestContext";
+
+const outputDir = path.join(__dirname, ".output");
+
+export interface IModelParams {
+  numElements: number;
+  fedGuids: boolean;
+}
+
+// TODO: dedup with other packages
+// for testing purposes only, based on SetToStandalone.ts, force a snapshot to mimic a standalone iModel
+export function setToStandalone(iModelPath: string) {
+  const nativeDb = new IModelHost.platform.DgnDb();
+  nativeDb.openIModel(iModelPath, OpenMode.ReadWrite);
+  nativeDb.setITwinId(Guid.empty); // empty iTwinId means "standalone"
+  nativeDb.saveChanges(); // save change to iTwinId
+  nativeDb.deleteAllTxns(); // necessary before resetting briefcaseId
+  nativeDb.resetBriefcaseId(BriefcaseIdValue.Unassigned); // standalone iModels should always have BriefcaseId unassigned
+  nativeDb.saveLocalValue("StandaloneEdit", JSON.stringify({ txns: true }));
+  nativeDb.saveChanges(); // save change to briefcaseId
+  nativeDb.closeIModel();
+}
+
+export function generateTestIModel(iModelParam: IModelParams): TestIModel {
```

Comment:
Should I just use this function in the rawInserts test, and just set a timer on it, so we don't need essentially the same code in 2 places? I don't believe there's many differences. I think maybe just adding a file name into that iModelParams interface

Reply:
sounds ideal. Not sure where you need a timer though?

Reply:
looks like there's a issue with the return type here for `generateTestIModel`, so maybe I can't do this, or its least not as simply. I didn't realize the sourceDb created from this gets used throughout the rest of the insert tests

Reply:
refactor this to return an `IModelDb` and wrap the `IModelDb` in a `TestModel` in the caller that needs it

Reply:
done